HIPStream.cu

```diff
@@ -0,0 +1,215 @@
+#include "hip/hip_runtime.h"
```

Comment:
Sure, please see the change in this [commit](https://github.com/UoB-HPC/GPU-STREAM/pull/10/commits/300ccd01d968330de0fe309501c39a167c717b14).